src/watcher.js

```diff
@@ -17,19 +18,21 @@ if (process.argv.length < 3) {
 
 const config = require(path.join('../config/', process.argv[2]))
 
+config.deployedBridgesRedisKey = process.env.DEPLOYED_BRIDGES_REDIS_KEY || 'deployed:bridges'
+const concurrency = process.env.MULTIPLE_BRIDGES_CONCURRENCY || 1
```

Comment:
Since `config.deployedBridgesRedisKey` and `concurrency` are only used on `erc-erc-multiple`, does it makes sense to move this definition into `base.config.js` on `bridgeMapperConfig`?